scripts/hubspot-helper.js

```diff
@@ -9,7 +52,19 @@ export default function loadHubSpot() {
   hsScriptEl.setAttribute('id', 'hs-script-loader');
   hsScriptEl.src = 'https://js.hsforms.net/forms/embed/v2.js';
   document.querySelector('head').append(hsScriptEl);
+
   hsScriptEl.addEventListener('load', () => {
-    buildForms(hbspt); // eslint-disable-line
+    // Wait for HubSpot's hbspt object to be available
+    const checkHubSpotReady = setInterval(() => {
+      if (window.hbspt) {
+        clearInterval(checkHubSpotReady);
+        initializeForms();
+      }
+    }, 100);
+
+    // Set a timeout to prevent infinite checking
+    setTimeout(() => {
+      clearInterval(checkHubSpotReady);
+    }, 10000);
```

Comment:
Is this 10s going to be enough time in all cases? It might be, but sometimes I don't see the form load.